test/index.spec.js

```diff
@@ -2,7 +2,7 @@
 
 const babelPresetEnv = require("../lib/index.js");
 const assert = require("assert");
-const { versions: electronToChromiumData } = require("electron-to-chromium");
+const electronToChromiumData = require("electron-to-chromium");
```

Comment:
Perhaps you can do

`const electronToChromiumVersions = require("electron-to-chromium/full-versions");`

Since you're not using other data (This keeps it a little smaller)

Reply:
需要好好学习了，特么的兴奋看到你是怎么做的？

Reply:
@Asudhaishao Хто його зна..